build/tizen/CMakeLists.txt

```diff
@@ -37,14 +37,39 @@ IF(NOT OPENCV_PKG_FOUND)
   SET(OPENCV opencv4 )
 ENDIF()
 
+OPTION(USD_LOADER_ENABLED "Whether dali-usd-loader is enabled." OFF)
```

Comment:
No need having it as an option as it's overridden later on anyway. May as well just use a variable.

Reply:
Thank you for your reply. Fix done.